vehicle/OVMS.V3/components/vehicle_mgev/src/mg_can_handler.cpp

```diff
@@ -131,14 +131,14 @@ void OvmsVehicleMgEv::IncomingPollFrame(CAN_frame_t* frame)
     }
     if (frame->MsgID == (gwmId | rxFlag) && (frameType & 4))
     {
-        ESP_LOGV(TAG, "Got an gateway authentication response");
-        GwmAuthentication(frame->origin, frameType & 3, data);
+        ESP_LOGV(TAG, "Got a gateway authentication response");
+        GwmAuthentication(frame->origin, frameType & 3, &data[1]);
```

Comment:
Without this change the frame type above can be accessed.

Reply:
Yep, or we can keep `data` here and correct the `GwmAuthentication()` function itself.

Reply:
It avoids the issues with incompatibility with the rest of the system

Reply:
Ok I'll adjust data within `GwmAuthentication()` itself.